intellijVersions.properties

```diff
@@ -1,5 +1,5 @@
 #Wed May 18 11:15:19 CEST 2022
-latestMinorsOfOldSupportedMajors=2020.3.4,2021.1.3,2021.2.4,2021.3.3
+latestMinorsOfOldSupportedMajors=2022.1.3
```

Comment:
(if `2022.1` stays as `earliestSupportedMajor`, then `latestMinorsOfOldSupportedMajors` should be empty)